source/server/management/performance-tweaks.rst

```diff
@@ -0,0 +1,91 @@
+==================
+Performance Tweaks
+==================
+
+Sponge provides several performance enhancing and tweaking options to help you with running a lag-free server, even
+under heavy load.
+
+.. note::
+  While we try to improve the performance of every server, the performance gained depends on your setup. Please run some
+  tests on your server to ensure you configured your Sponge server to get the most out of it!
+
+SpongeForge only enhancements
+=============================
+
+Entity activation range
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+
+  entity-activation-range {
```

Comment:
This is actually [available in SpongeForge and SpongeVanilla](https://github.com/SpongePowered/SpongeCommon/commit/fbdedf182772129f1d1d147221c30320d607fe42) now.